### DISCUSSION TITLE
[2688680] 魔塔少女 [本体]

### DISCUSSION THREAD

**Discussion by wujcwin** [pjy612/SteamManifestCache#550] [Share]
`appid:2688680`
keys:
```
2688681;ac1d0bebb6bb797c85f3e0f3b1de659db587290ebb828f9194ef70ff4c123e5d
```
[2688681_6012998768251508802.zip](https://github.com/pjy612/SteamManifestCache/files/14716596/2688681_6012998768251508802.zip)

更新了下清单 话说留邮箱是干啥用的？
————————————————————————
3.22更新

**Comment by pjy612** [comment]
> 更新了下清单 话说留邮箱是干啥用的？

失效了的话 给人发邮件更新？（bushi）

**Reply by wujcwin** [reply]
好像也挺合理 :)

**Reply by pjy612** [reply]
感谢分享 有缘再见
